### ISSUE TITLE
Zoom landscape image not working on split pages

### ISSUE THREAD

**Issue by ghostbear** [tachiyomiorg/tachiyomi#6643]
### Steps to reproduce

1. Swipe next onto split page

### Expected behavior

Zoom in when changing page

### Actual behavior

https://user-images.githubusercontent.com/6576096/153766680-b5a06014-77df-40ee-9beb-8a21712deda8.mp4

### Tachiyomi version

r4159

### Android version

Android 11

### Device

OnePlus 8

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.13.1](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
I don't quite understand what's happening on the video. It seems it's working, but only once the image has been loaded once?
